cfme/tests/configure/test_register_appliance.py

```diff
@@ -71,13 +72,36 @@ def pytest_generate_tests(metafunc):
 
 
 @pytest.fixture(scope="function")
-def appliance_preupdate(temp_appliance_preconfig_funcscope, appliance):
-    """Requests appliance from sprout and configures rpms for crud update"""
+def appliance_preupdate(temp_appliance_preconfig_funcscope):
+    """Requests appliance from sprout and configures rpms for crud update
+
+    To create the required repo (fresh-new (clean) CFME appliance required).
+
+    register and attach the system to RHN
+    # SERVER=...redhat.com
+    # yum --downloadonly --downloaddir /therepo install -y rpm-build
+    # yum install -y rpm-build createrepo
+    # rpmbuild  --rebuild http://$SERVER/~jhenner/rpmrebuild/rpmrebuild-2.14-1.src.rpm
+    # cp rpmbuild/RPMS/noarch/rpmrebuild-2.14-1.noarch.rpm /therepo
+    # createrepo /therepo
+
+    # curl 'http://$SERVER/~jhenner/rpmrebuild_repo/5.XX/rpmrebuild.repo' \
+        -o /therepo/rpmrebuild.repo
+    # vim /therepo/rpmrebuild.repo
+    # rsync -avP /therepo/ jhenner@$SERVER:public_html/rpmrebuild_repo/5.XX
+    """
+    appliance = temp_appliance_preconfig_funcscope
+
     try:
-        url = cfme_data['basic_info']['rpmrebuild_59']
+        url = VersionPicker(
+            {'5.10': cfme_data['basic_info']['rpmrebuild_510'],
+             '5.11': cfme_data['basic_info']['rpmrebuild_511']}).pick(appliance.version)
```

Comment:
_Optional_ since you already have the `appliance` object handy you can just use that. 
```suggestion
        if appliance.version < '5.11':
            url = cfme_data['basic_info']['rpmrebuild_510']
        else:
            url = cfme_data['basic_info']['rpmrebuild_511']
```

Reply:
I think I would like to keep it as it is now.